classes/email/prepare.php

```diff
@@ -6,112 +6,122 @@
  * Methods for preparing email address strings.
  * Useful when dealing with strings that may be in the rfc822 format -- "name <email@example.com>"
  */
-class Caldera_Forms_Email_Prepare{
```

Comment:
@robwilde What changed here? I can't tell since everything changed. Can you please highlight the change you made?

Reply:
@Shelob9 updated the lines below of the Caldera_Forms_Email_Prepare. Updating the doc blocks now

Reply:
I use the built in code reformat to set everything to WP layout, this may have caused it not to pickup on the minor adjustments